NAMESPACE

```diff
@@ -39,5 +39,6 @@ importFrom(rlang,abort)
 importFrom(rlang,inform)
 importFrom(rlang,warn)
 importFrom(tibble,tibble)
+importFrom(tidyr,pivot_longer)
```

Comment:
I chose to export this, since `pivot_wider` is already exported, and the names are pretty unique to `tidyr`. Also happy to go the other direction, and explicitly namespace all `pivot_*`s